packages/pangea-sdk/pangea/verify_audit.py

```diff
@@ -201,23 +213,26 @@ def comma_sep(s: Set):
     if pending_roots:
         logger.debug(f"Published root(s) {comma_sep(pending_roots)} could not be fetched from Arweave")
 
-    if pending_roots and audit:
-        # and then Pangea (if we've set an audit client)
-        try:
-            logger.debug(f"Fetching root(s) {comma_sep(pending_roots)} from Pangea")
-            pangea_roots |= {int(k): v for k, v in get_pangea_roots(pending_roots).items()}  # type: ignore[operator]
-            update_pending_roots()
-            status = Status.SKIPPED
-        except:
-            pass
-
-        if pending_roots:
-            logger.debug(f"Roots {comma_sep(pending_roots)} could not be fetched")
+    if pending_roots:
+        if audit:
+            # and then Pangea (if we've set an audit client)
+            try:
+                logger.debug(f"Fetching root(s) {comma_sep(pending_roots)} from Pangea")
+                pangea_roots |= {int(k): v for k, v in get_pangea_roots(tree_name, pending_roots).items()}  # type: ignore[operator]
+                update_pending_roots()
+                status = Status.SUCCEEDED_PANGEA
+            except:
+                pass
+
+            if pending_roots:
+                logger.debug(f"Roots {comma_sep(pending_roots)} could not be fetched")
+        else:
+            logger.debug("Set PANGEA_DOMAIN and PANGEA_TOKEN envvars to fetch roots from Pangea")
```

Comment:
Based on:
```
    parser.add_argument(
        "--token", "-t", default=os.getenv("PANGEA_TOKEN"), help="Pangea token (default: env PANGEA_TOKEN)"
    )
    parser.add_argument(
        "--domain", "-d", default=os.getenv("PANGEA_DOMAIN"), help="Pangea domain (default: env PANGEA_DOMAIN)"
    )
    ```
  
They have the option provide a token on args too...

Reply:
true, changing